src/reportSources/legacyFflogs/__tests__/eventAdapter.ts

```diff
@@ -11,7 +11,7 @@ import {ReassignUnknownActorStep} from '../eventAdapter/reassignUnknownActor'
 // TODO: If we need to do this >1 times, work out a cleaner way of doing this.
 jest.mock('../eventAdapter/reassignUnknownActor')
 type ReassignUnknownActorStepParams = ConstructorParameters<typeof ReassignUnknownActorStep>
-const MockReassignUnknownActorStep = ReassignUnknownActorStep as jest.Mock<ReassignUnknownActorStep>
+const MockReassignUnknownActorStep = ReassignUnknownActorStep as unknown as jest.Mock<ReassignUnknownActorStep>
```

Comment:
i mean tests so who cares, but what was the error?

Reply:
It was a can't cast typescript error that specified "cast to unknown before casting to the Mock".  The cast for injectable.constructor in the Parser class was the same error, but I inferred from other similar code and the context of that code that changing to injectable.constructor seemed reasonable, and that didn't break anything else that I found.

Reply:
Yeah the other spot was probably a bug. Oops.